license/header_check.sh

```diff
@@ -1,17 +0,0 @@
-#
-# Copyright (c) 2015 Intel Corporation
-# 
-# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
-# in compliance with the License. You may obtain a copy of the License at
-# 
-# http://www.apache.org/licenses/LICENSE-2.0
-# 
-# Unless required by applicable law or agreed to in writing, software distributed under the License
-# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
-# or implied. See the License for the specific language governing permissions and limitations under
-# the License.
-# 
-
-cd ..
-mvn license:check
-exit $?
```

Comment:
This script can't be removed. We release our software under Apache License and we are obliged to have a proper header in all files. This script checks this and prevents from building project without those headers.